flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/reconciler/SessionReconciler.java

```diff
@@ -57,13 +57,14 @@ public void reconcile(FlinkDeployment flinkApp, Context context, Configuration e
             flinkApp.getStatus()
                     .setJobManagerDeploymentStatus(JobManagerDeploymentStatus.DEPLOYING);
             IngressUtils.updateIngressRules(flinkApp, effectiveConfig, kubernetesClient);
+        } else {
```

Comment:
Compared with else, I prefer a `return` at the end if `if...`

Reply:
Fixed